admin/src/components/CreationFormular.vue

```diff
@@ -140,78 +135,38 @@ export default {
     updateRadioSelected(name) {
       // do we want to reset the memo everytime the month changes?
       this.text = this.$t('creation_form.creation_for') + ' ' + name.short + ' ' + name.year
-      if (this.type === 'singleCreation') {
-        this.rangeMin = 0
-        this.rangeMax = name.creation
-      }
+      this.rangeMin = 0
+      this.rangeMax = name.creation
     },
     submitCreation() {
-      let submitObj = []
-      if (this.type === 'massCreation') {
-        this.items.forEach((item) => {
-          submitObj.push({
-            email: item.email,
+      this.$apollo
+        .mutate({
+          mutation: adminCreateContribution,
+          variables: {
+            email: this.item.email,
             creationDate: this.selected.date,
             amount: Number(this.value),
             memo: this.text,
-          })
+          },
+        })
+        .then((result) => {
+          this.$emit('update-user-data', this.item, result.data.adminCreateContribution)
+          this.$store.commit('openCreationsPlus', 1)
+          this.toastSuccess(
+            this.$t('creation_form.toasted', {
+              value: this.value,
+              email: this.item.email,
+            }),
+          )
+          // what is this? Tests says that this.text is not reseted
+          this.$refs.creationForm.reset()
```

Comment:
Is this a ToDo or what should this comment express?

Reply:
Out of scope of this PR. This should be addressed to @oderly